### ISSUE TITLE
[APP SUBMITTED]: No JSON object could be decoded

### ISSUE THREAD

**Issue by yeyeoke** [SiCKRAGETV/sickrage-issues#1823]
### INFO
Python Version: **2.7.9 (default, Dec 10 2014, 12:24:55) [MSC v.1500 32 bit (Intel)]**
Operating System: **Windows-8-6.2.9200**
Branch: **master**
Commit: SiCKRAGETV/SickRage@e6c4cfd6546b74d5527361f6f0bdc3adc232bdb0
Link to Log: https://gist.github.com/e69a65e77dbec29238f0
### ERROR
```
SEARCHQUEUE-MANUAL-277165 :: [TorrentDay] :: Error while searching TorrentDay, skipping: No JSON object could be decoded
```
---
_STAFF NOTIFIED_: @SiCKRAGETV/owners @SiCKRAGETV/moderators

**Comment by yeyeoke** [comment]
This is what I get when I try to search a show on TorrentDay. TorrentDay is my only provider and I haven't changed anything. The issue just started a couple of days ago.

**Comment by fernandog** [comment]
@yeyeoke disable proxy for TD

**Comment by fernandog** [comment]
@miigotu can we apply the same code from RARBG in TD?

**Comment by yeyeoke** [comment]
No proxy enabled. Tried it turned on aswell but still not working.

**Comment by fernandog** [comment]
@yeyeoke what about global proxy?

**Comment by yeyeoke** [comment]
@fernandog Not sure what you mean by global proxy but I don't have any proxy config in router or otherwise, Nor in my PC.

**Comment by fernandog** [comment]
@yeyeoke go to log page and use the search input to search for: search page URL

try to open TD url in your browser

**Comment by yeyeoke** [comment]
@fernandog That search doesnt return anything. Tried different variations of URL and I don't seem to get anything. I tried turning debug on aswell but still nothing. I can open the regular TD-url in my browser.

**Comment by fernandog** [comment]
Please post SR TD url search here

**Comment by yeyeoke** [comment]
https://tdonline.org/V3/API/API.php

**Comment by miigotu** [comment]
Thats not the full url.

**Comment by fernandog** [comment]
@yeyeoke TD is working here:

```
2015-06-15 13:18:50 DEBUG    SEARCHQUEUE-MANUAL-274897 :: [TorrentDay] :: Existing episode status is wanted/unaired/skipped, getting found episode
2015-06-15 13:18:50 DEBUG    SEARCHQUEUE-MANUAL-274897 :: [TorrentDay] :: Existing episode status: 3 (Wanted)
2015-06-15 13:18:50 DEBUG    SEARCHQUEUE-MANUAL-274897 :: [TorrentDay] :: Any,Best = [ 1080p WEB-DL ] [ 1080p WEB-DL ] Found = [ 1080p WEB-DL ]
```

**Comment by yeyeoke** [comment]
@fernandog Damn it, was hoping they had changed their API or something. Still doesnt work for me. The URL, should it be under "info" in the log?

**Comment by fernandog** [comment]
@yeyeoke no. debug

**Comment by fernandog** [comment]
@yeyeoke did you added TD as a custom torrent provider or did you enabled the TD in  the list of providers??

**Comment by yeyeoke** [comment]
@fernandog I just enabled it in the list of providers

**Comment by fernandog** [comment]
@yeyeoke did you find the full search url of TD? please post here

**Comment by yeyeoke** [comment]
@fernandog Yeah, its weird because I've tried searching but I can't find it. If i enable torrentleech it shows the search url but not for torrentday.

**Comment by fernandog** [comment]
Search do Torrentday instead

**Comment by yeyeoke** [comment]
@fernandog Tried aswell but the only time i get an url is when the proxy option is enabled. then i get: "Proxified URL: http://proxite.eu/browse.php?u=https%3A%2F%2Ftdonline.org%2FV3%2FAPI%2FAPI.php&b=32&f=norefer"

**Comment by miigotu** [comment]
Turn on debug.

**Comment by fernandog** [comment]
ops!

**Comment by yeyeoke** [comment]
This is kind of weird. I reinstalled Sickrage (deleted and pulled all files from git), I turned on debug and it still show nothing similar to a url.

**Comment by Sandrus** [comment]
### INFO
Python Version: **2.7.10 (default, Jun  3 2015, 20:43:13) [GCC 4.7.3 20130102 (prerelease)]**
Operating System: **Linux-3.10.35-x86_64-with-glibc2.2.5**
Locale: unknown
Branch: **master**
Commit: SiCKRAGETV/SickRage@e6c4cfd6546b74d5527361f6f0bdc3adc232bdb0
Link to Log: https://gist.github.com/1c9a785577553933e099
### ERROR
```
SEARCHQUEUE-BACKLOG-273690 :: [T411] :: Error while searching T411, skipping: No JSON object could be decoded
```
---
_STAFF NOTIFIED_: @SiCKRAGETV/owners @SiCKRAGETV/moderators

**Comment by fernandog** [comment]
Did you changed log level to debug?

**Comment by Sandrus** [comment]
Yes

**Comment by miigotu** [comment]
Ugh, so how many providers do I need to fix this in? lol Why wasn't this all reported before, when I was fixing rarbg xD.

**Comment by Sandrus** [comment]
Sorry for that, let's hope that T411 is the last one ^^

**Comment by miigotu** [comment]
Have you tried other proxy urls? I wont be able to fix proxying via proxies who cannot handle https with tdonline and t411 because these providers don't use api, and require us to use https.

If they do not work with any proxy, I cannot fix it without an account, so the best bet I guess is to disable proxy. Proxy is ridiculous for sites using https anyways unless you are blocked by your isp from access to that site.

**Comment by Rouzax** [comment]
I have the same issue https://github.com/SiCKRAGETV/sickrage-issues/issues/1850

**Comment by Sandrus** [comment]
@miigotu T411 didn't work with proxy, i have tried the 4 and get the same error.
I will remove the option, no big deal.
Thanks.

**Comment by Rouzax** [comment]
I don't use a proxy to access torrentday and have the same issue.

**Comment by Rouzax** [comment]
Let me know if you need anything.

**Comment by miigotu** [comment]
Committed a fix in SiCKRAGETV/SickRage#1978 for Torrentday not working even without proxy, for now proxy still probably won't work.

**Comment by Rouzax** [comment]
@miigotu 
Although I'm running the Master Git branch  SiCKRAGETV/SickRage@52a6c31cdf545c4e2fd22925a92a3d0824c429cf TorrentDay is working again, without any changes from my end.
